bridgetown-core/lib/bridgetown-core/configuration.rb

```diff
@@ -226,7 +226,7 @@ def read_config_file(file)
         raise ArgumentError, "Configuration file: (INVALID) #{file}".yellow
       end
 
-      Bridgetown.logger.info "Configuration file:", file
+      Bridgetown.logger.debug "Configuration file:", file
```

Comment:
So by changing this from info to debug, it means that by default the configuration file line is never output to the user's terminal (they'd have to pass the --verbose flag).￼ I'm not necessarily opposed to this, but it's an obvious change from previous behavior.

Reply:
My thinking here was that it's not really super important information so I reckon it's ok to bump it down a level. The reason I did it now is because it gets logged out when creating a new site. This is because the `webpack` command creates a configuration object that reads the file. 

This was the reason I inadvertently turned off logging earlier; I was trying to get this one line to stop being logged out when `bridgetown new` is run.

Do you think it's something that we absolutely should be logging out without the verbose flag?

Reply:
I've found it annoying in some of the work I've done with SSR prototypes as well. Let's do it and just make a mention in our changelog/release notes. :+1: